<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>车载系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
</string>
   </property>
   <widget class="QStackedWidget" name="stack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="index">
     <property name="styleSheet">
      <string notr="true">#index{
	border-image: url(:/imgs/bg.jpeg);
}</string>
     </property>
     <widget class="QLabel" name="timeLab">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>300</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: none;
color: rgb(255, 255, 255);
font: 40pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>10:20:20</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="dateLab">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>200</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: none;
color: rgb(255, 255, 255);
font: 14pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>2025-03-23 周二</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="weatherBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>400</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/imgs/weather.png);

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="musicBtn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>400</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
border-image: url(:/imgs/music.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="videoBtn">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>400</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
border-image: url(:/imgs/video.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="locationBtn">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>400</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
border-image: url(:/imgs/location.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
